config/04-kubernetes/42-hub-preview-redirect-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ui
+  namespace: tekton-hub-preview
+  annotations:
+    acme.cert-manager.io/http01-edit-in-place: 'true'
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/permanent-redirect: https://hub.tekton.dev
+    nginx.ingress.kubernetes.io/force-ssl-redirect: 'true'
+spec:
+  tls:
+    - secretName: ui-hub-preview-tekton-dev-tls
+      hosts: [hub-preview.tekton.dev]
+  rules:
```

Comment:
I'm not sure this section is needed at all since we will redirect to hub in all cases, but I don't know if the ingress will be happy without it